### DISCUSSION TITLE
Sign commits using SSH key

### DISCUSSION THREAD

**Discussion by mirsafari** [NeogitOrg/neogit#1274] [Q&A]
Hello. 

Following the GitHub docs [here](
https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key) I've configured signing commits using SSH key. Everything works fined when using `git` CLI (`git commit -S -m ""`). 

I struggle to find documentation on how to use this with NeoGit. There is an option `=S Sing using gpg` but it is using different flag `--gpg-sign=` and I do not know how to use this functionality with SSH. 

Any help is appreciated. 
Thanks in advance!

**Comment by CKolkey** [comment]
I've added better docs for this in `nightly`: https://github.com/NeogitOrg/neogit/blob/nightly/doc/neogit.txt#L230-L246

Lemme know if that addresses your question :)

**Reply by mirsafari** [reply]
Hi. Thanks for the reply. Unfortunately not as I use SSH (`git config --global gpg.format ssh`) for signing, the way it is indicated in the docs.

**Reply by clobrano** [reply]
I have the same problem. Using `git commit -S` works just fine, but NeoGit enforces to add the `--gpg-sign=`

**Reply by CKolkey** [reply]
Does this setup work? Not the 1password part, but the git config

https://developer.1password.com/docs/ssh/git-commit-signing/#step-1-configure-git-commit-signing-with-ssh

**Reply by clobrano** [reply]
Do you mean
```
Set commit.gpgsign to true so you don't need to include the -S flag with each commit. (optional)
```

I think it will solve my problem yes, thanks
